GT.sln

```diff
@@ -1,24 +1,21 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30114.105
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31912.275
```

Comment:
Why was this step necessary?

Reply:
May be a result of updating VS and SSMS/Azure Studio and inherently updating the sln. Nothing done intentionally nor manually on my part.

Reply:
If so we should keep the updated lines. Resolving comment.